journals/apps/core/management/commands/create_site.py

```diff
@@ -141,7 +149,9 @@ def build_oauth_settings(
             "SOCIAL_AUTH_EDX_OIDC_URL_ROOT": urljoin(lms_url_root, '/oauth2'),
             "SOCIAL_AUTH_EDX_OIDC_KEY": client_id,
             "SOCIAL_AUTH_EDX_OIDC_PUBLIC_URL_ROOT": urljoin(lms_public_url_root_override, 'oauth2'),
-            "SOCIAL_AUTH_EDX_OIDC_LOGOUT_URL": urljoin(lms_public_url_root_override, 'logout')
+            "SOCIAL_AUTH_EDX_OIDC_LOGOUT_URL": urljoin(lms_public_url_root_override, 'logout'),
+            "SOCIAL_AUTH_EDX_OIDC_ISSUER": urljoin(lms_url_root, '/oauth2'),
+            "SOCIAL_AUTH_EDX_OIDC_ISSUERS": [lms_url_root],
```

Comment:
The issuer is `root/oauth2` and the issuers is just `root`?

Reply:
@tuchfarber 
yes, seems odd but that's what the provision.sql had so I copied it, and it seems to work